soliket/tests/test_runs.py

```diff
@@ -5,8 +5,8 @@
 from cobaya.run import run
 
 
-@pytest.mark.parametrize("lhood", ["mflike", "lensing", "lensing_lite", "multi"])
-def test_evaluate(lhood):
+@pytest.mark.parametrize("lhood", ["mflike", "lensing", "lensing_lite", "multi, cross_correlation"])
+def test_run(lhood):
```

Comment:
This should be changed back to `test_evaluate`

Reply:
And below, please add `cross_correlation` to the `test_mcmc` parameter list as well.